documentation/plain_rules.md

```diff
@@ -106,7 +106,7 @@ Sends a SMS to a number set as an action paramter with the body of the message b
 ```
 The field `parameters` include a field `to` with the number to send the message to.
 
-The `template` field performs [attribute substitution](#string-substitution-syntax).
+The `template` and `to` fields perform [attribute substitution](#string-substitution-syntax).
```

Comment:
I understand that to keep documentation coherent, the list just under "This substitution can be used in the the following fields" at "String substitution syntax" should be also modified.